.github/actionlint.yaml

```diff
@@ -2,6 +2,7 @@ self-hosted-runner:
   # Labels of self-hosted runner in array of strings.
   labels:
     - kic-plus
```

Comment:
is this required? If yes then can it be var which can be used across all workflows?
```
runner:
        type: string
        default: <here>
```
or
```
runs-on: <here>
```
or
```
runner: <here>
```
so we need to update the value once